doc/rst/usingchapel/executing.rst

```diff
@@ -348,6 +348,27 @@ executable.
                    warnings that are printed by default.
 
 
+.. _oversubscribed-execution:
+
+----------------
+Oversubscription
+----------------
+
+In multi-locale Chapel executions programs can run "oversubscribed",
+with more than one Chapel locale per system compute node.  Both the
+``gasnet`` and ``ofi`` communication layers support oversubscription.
+When running oversubscribed, performance may be improved by setting
+the following environment variable:
```

Comment:
A naive user might interpret this at thinking they can get good performance by running oversubscribed but we're really just trying to limit the huge performance hit from doing so (it'll be very hard to find a program that runs faster with -nl2 oversubscribed vs -nl1.

I might reword this paragraph to say something more like "To simulate running on multiple locales Chapel programs can run oversubscribed, running multiple chapel locales per compute node. Both gasnet and ofi... <whatever>. For performance in multi-locale Chapel assumes it owns the entire node and will do things like spinwaiting or setting up processor affinity. These help performance when Chapel is the only thing running on a node, but can significantly deteriorate performance when multiple Chapel instances are competing for the same resources." (Actual phrasing and wording aren't great, but I would try to convey that we're just limiting the performance overhead for running oversubscribed, you'll never get "good performance" when running oversubscribed.

Reply:
Reworked it ...